### PR TITLE
adding readme to rethinkdb input

### PR DESCRIPTION
**Pull request by russorat**
### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [ ] Associated README.md updated.
- [ ] Has appropriate unit tests.

### CODE REVIEW

File: plugins/inputs/rethinkdb/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+```toml
+[[inputs.rethinkdb]]
+  ## An array of URI to gather stats about. Specify an ip or hostname
+  ## with optional port add password. ie,
+  ##   rethinkdb://user:auth_key@10.10.3.30:28105,
+  ##   rethinkdb://10.10.3.33:18832,
+  ##   10.0.0.1:10000, etc.
+  servers = ["127.0.0.1:28015"]
+  ##
```
Comment (glinton) [inline plugins/inputs/rethinkdb/README.md]: Remove these empty line comments.

File: plugins/inputs/rethinkdb/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  # servers = ["rethinkdb2://username:password@127.0.0.1:28015"]
+  ##
+  ## If you use older versions of rethinkdb (<2.2) with auth_key, protocol
+  ## have to be named "rethinkdb".
+  # servers = ["rethinkdb://username:auth_key@127.0.0.1:28015"]
+```
+
+### Metrics
+
+- rethinkdb
```
Comment (glinton) [inline plugins/inputs/rethinkdb/README.md]: Indentation and new formatting:

```md
- measurement1
  - tags:
    - tag1 (optional description)
    - tag2
  - fields:
    - field1 (type, unit)
    - field2 (float, percent)
```

File: plugins/inputs/rethinkdb/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    - disk_read_bytes_per_sec (integer, reads)
+    - disk_read_bytes_total (integer, bytes)
+    - disk_written_bytes_per_sec (integer, bytes)
+    - disk_written_bytes_total (integer, bytes)
+    - disk_usage_data_bytes (integer, bytes)
+    - disk_usage_garbage_bytes (integer, bytes)
+    - disk_usage_metadata_bytes (integer, bytes)
+    - disk_usage_preallocated_bytes (integer, bytes)
+  
+- rethinkdb_engine
```
Comment (glinton) [inline plugins/inputs/rethinkdb/README.md]: Something we can do before merging, but lists in github markdown should alternate between `-` and `+` to preserve spacing:
```md
- rethinkdb

+ rethinkdb_engine
```

Comment (danielnelson) [reply plugins/inputs/rethinkdb/README.md]: I'll change this after merge.

**Pull request merged**